target/linux/ath79/image/generic.mk

```diff
@@ -1526,6 +1526,24 @@ define Device/ocedo_ursus
 endef
 TARGET_DEVICES += ocedo_ursus
 
+define Device/openmesh_om2p-common
+  DEVICE_VENDOR := OpenMesh
+  DEVICE_MODEL := OM2P
+  DEVICE_PACKAGES := uboot-envtools
+  IMAGE_SIZE := 7168k
+  BLOCKSIZE := 256k
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma | pad-to $$(BLOCKSIZE)
+  IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | openmesh-image ce_type=OM2P | append-metadata
```

Comment:
Please wrap these lines to have max. 74 chars in one line.
Hanging lines with one tab for indent ...

Reply:
Consider this done